test/px-dropdown-custom-tests.js

```diff
@@ -421,6 +421,37 @@ describe("Disabled element tests for px-dropdown", function(done) {
     done();
   });
 });
+describe("Read Only item tests for px-dropdown", function(done) {
```

Comment:
Do we also need test cases for `disabled-viewable`? And the combination of `read-only` and `disabled-viewable`?

Reply:
i have added a couple of more tests to validate the different px-dropdown states of disabled, read-only and disabled-viewable